modxcms/revolution
# [3.x]-Follow up element lexicon and style changes for Element/Resource form panels

### What does it do?
Removes a handful of unused lexicon entries and changes Resource TV checkboxes to normal checkbox style.

### Why is it needed?
This is just some housekeeping after the recent merge of my 6-part "Fix Various Issues with Form Panel Validation and Field Display" PRs. Also fixes an update I missed to the Snippet editing panel's first tab (making it read "General Information" instead of "Create/edit snippet").

### How to test
Rebuild the manager template via `grunt build` and take a quick look at the Resource and Element panels to verify changes.

### Related issue(s)/PR(s)
None
